### ISSUE TITLE
v1.1 arrived

### ISSUE THREAD

**Issue by gkasprow** [sinara-hw/HV_AMP_8CH#4]
![2021-06-02 20 04 41](https://user-images.githubusercontent.com/4325054/120891923-8d26c400-c60b-11eb-86d6-54be43bfec21.jpg)

**Comment by jordens** [comment]
Is that with apexes? Could you upload the release data and PDF for that? I'd like to follow this development!

**Comment by gkasprow** [comment]
yep. Done https://github.com/sinara-hw/HV_AMP_8CH/releases/tag/v1.1

**Comment by gkasprow** [comment]
Full power BW
![tek00008](https://user-images.githubusercontent.com/4325054/122832845-44019000-d2ec-11eb-917c-09bedfcd1986.png)

at 50kHz it's able to output 260Vpp
![tek00007](https://user-images.githubusercontent.com/4325054/122832897-5a0f5080-d2ec-11eb-9044-5a6d80284e6f.png)

at 100kHz it's able to output 124Vpp
![tek00006](https://user-images.githubusercontent.com/4325054/122832936-698e9980-d2ec-11eb-9e9d-b573ad19c6fa.png)

at 250kHz - 42Vpp
![tek00005](https://user-images.githubusercontent.com/4325054/122832968-757a5b80-d2ec-11eb-9fa5-6ea0e73d3a1c.png)

at 500kHz - 26Vpp, limited by SR
![tek00004](https://user-images.githubusercontent.com/4325054/122833008-86c36800-d2ec-11eb-9d7b-98da6dde3dca.png)

at 1MHz - 13Vpp limited by SR
![tek00003](https://user-images.githubusercontent.com/4325054/122833021-8c20b280-d2ec-11eb-962e-677d80d177c8.png)

square wave at 33kHz. We can clearly see SR of 30V/us as in the datasheet. It means that the input amp didn't break the spec. We need input amp due to HV amp GPB.
![tek00009](https://user-images.githubusercontent.com/4325054/122833186-d1dd7b00-d2ec-11eb-8c75-303ee8fd544d.png)

**Comment by gkasprow** [comment]
Thermal protection works. After heating the temp sensor the HV and 24V rails get disabled.
